README.md

```diff
@@ -21,26 +29,85 @@ loam.open(blob).then((dataset) => {
   dataset.width()
     .then((width) => /* do stuff with width */)
     .then(() => dataset.close()); // It's important to close datasets after you're done with them
-  // Available functions on dataset are:
-  // - width()
-  // - height()
-  // - transform() (returns in GDAL ordering, not affine transform ordering)
-  // - wkt()
-  // - count() (returns number of bands)
-  // - convert() (Takes an array of strings that match options to gdal_translate, e.g. ['-of', 'PNG'], returns dataset)
-  // - warp() (Takes an array of strings that match options to gdalwarp https://www.gdal.org/gdalwarp.html, returns dataset)
-
-});
-
-// Utility function for reprojecting points. srcSRS and destSRS must be full WKT strings.
-loam.reproject(srcSRS, destSRS, [[x, y], [x, y], ...]);
 ```
-Further examples are available in the tests.
-
-To install, make `loam.min.js` web-accessible or include it in your bundle. Then,
-make sure the following are web-accessible from the same path (but NOT included in
-your bundle):
-- loam-worker.min.js
-- [gdal.js](https://www.npmjs.com/package/gdal-js)
-- [gdal.wasm](https://www.npmjs.com/package/gdal-js)
-- [gdal.data](https://www.npmjs.com/package/gdal-js)
+
+## Functions
```

Comment:
❤️ the detail and completeness here and everything looks correct at a glance from someone who hasn't used loam in a while. 

Is there a way to space the content (the individual methods) out a bit? Maybe just personal preference so I don't feel strongly about it, but it's difficult to visually separate the methods, their arguments, and return types when I read through the page, probably because there's no additional space between methods. Struggling to find an example that does README docs similarly enough to make a useful comparison.

Reply:
Thanks, that's a good call. I based the formatting on the MDN docs, but they have the benefit of being able to design their own styling, whereas I just have to settle for what GH Markdown gives me. I ended up adding a manual `<br />` between each function section to space things out a bit, which is a little ugly  in the code but easy enough to copy-paste.